type_classes.md

```diff
@@ -705,19 +1007,22 @@ theorem ex : True ∧ 2 = 1+1 := by
 ```
 
 They work as follows. The expression ``decide p`` tries to infer a
-decision procedure for ``p``, and, if it is successful, evaluates to
+Decidable procedure for ``p``, and, if it is successful, evaluates to
 either ``true`` or ``false``. In particular, if ``p`` is a true closed
 expression, ``decide p`` will reduce definitionally to the Boolean ``true``.
 On the assumption that ``decide p = true`` holds, ``of_decide_eq_true``
 produces a proof of ``p``. The tactic ``decide`` puts it all together: to
 prove a target ``p``. By the previous observations,
-``decide`` will succeed any time the inferred decision procedure
+``decide`` will succeed any time the inferred Decidable procedure
  for ``c`` has enough information to evaluate, definitionally, to the ``isTrue`` case.
 
+BUGBUG: ``c`` is not defined? And where is ``isTrue`` mentioned in the code snippet?
```

Comment:
`isTrue` is one of the constructors of the `Decidable` type class mentioned when its definition is first introduced.

Reply:
Thanks